src/PreCharge/PreCharge.cpp

```diff
@@ -257,10 +276,13 @@ uint16_t PreCharge::getObjectDictionarySize() {
 }
 
 void PreCharge::sendChangePDO() {
```

Comment:
This method needs documentation in the `.hpp` file. Also, it should probably be made private.